integration-tests/src/test/java/tachyon/client/table/RawColumnIntegrationTest.java

```diff
@@ -26,42 +23,47 @@
 import tachyon.Constants;
 import tachyon.LocalTachyonClusterResource;
 import tachyon.TachyonURI;
-import tachyon.client.TachyonFS;
-import tachyon.client.TachyonFile;
+import tachyon.client.file.TachyonFile;
+import tachyon.client.file.TachyonFileSystem;
 import tachyon.conf.TachyonConf;
+import tachyon.thrift.FileInfo;
 
 /**
  * Integration tests for tachyon.client.RawColumn.
  */
+// TODO(calvin): Move this into TachyonRawTablesIntegrationTest
 public class RawColumnIntegrationTest {
 
   @Rule
   public LocalTachyonClusterResource mLocalTachyonClusterResource =
       new LocalTachyonClusterResource(10000, 1000, Constants.GB);
-  private TachyonFS mTfs = null;
+  private TachyonFileSystem mTachyonFileSystem = null;
+  private TachyonRawTables mTachyonRawTables = null;
 
   @Before
   public final void before() throws Exception {
-    mTfs = mLocalTachyonClusterResource.get().getOldClient();
+    mTachyonFileSystem = mLocalTachyonClusterResource.get().getClient();
```

Comment:
I see. I don't think `TachyonFileSystem` is necessary to be obtained from the cluster. But this is beyond the scope of this PR.